### ISSUE TITLE
unspent utxo

### ISSUE THREAD

**Issue by ygcool** [team-exor/eiquidus#35]
Hi,
Does it support querying the unspent utxo list of any address?

**Comment by joeuhren** [comment]
No, the explorer does not yet keep track of unspent UTXO data although it has been briefly talked about [here](https://github.com/team-exor/eiquidus/issues/24).

Feel free to review the public explorer [here](https://explorer.exor.io) to get a feel for how it currently works.

**Comment by regtable** [comment]
+1 this would be a neat function so it could be used as similar to an electum server
